source/addonHandler/__init__.py

```diff
@@ -210,23 +229,27 @@ def _getAvailableAddonsFromPath(path):
 					yield a
 				except:
 					log.error("Error loading Addon from path: %s", addon_path, exc_info=True)
-				
+
 _availableAddons = collections.OrderedDict()
-def getAvailableAddons(refresh=False, filterFunc=None):
+
+
+def getAvailableAddons(refresh=False, filterFunc=None, isFirstLoad=False):

```

Comment:
Could the parameter and return types be updated here - at least for the new variable.

Reply:
Done as of cae49fbd2